src/snowflake/snowpark/mock/constants.py

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+SUPPORTED_CSV_READ_OPTIONS = (
```

Comment:
NIT: Could we put it in `mock/file_operation.py` instead? I prefer having only one `constants.py`.

Reply:
sure, I have no preference where it should be placed